Makefile

```diff
@@ -646,18 +646,20 @@ test-failure: $(failure_tests:=.run-expected)
 
 # kevm_pyk Tests
 
-kevm_pyk_tests :=                                                         \
-                  tests/interactive/vmLogTest/log3.json.gst-to-kore.check \
-                  tests/foundry/kompiled/timestamp                        \
-                  tests/foundry/foundry.k.check                           \
-                  tests/specs/bihu/functional-spec.k.prove                \
-                  tests/specs/examples/empty-bin-runtime.k                \
-                  tests/specs/examples/erc20-bin-runtime.k                \
+kevm_pyk_tests :=                                                                                              \
+                  tests/interactive/vmLogTest/log3.json.gst-to-kore.check                                      \
+                  tests/ethereum-tests/BlockchainTests/GeneralStateTests/VMTests/vmArithmeticTest/add.json.run \
+                  tests/foundry/kompiled/timestamp                                                             \
+                  tests/foundry/foundry.k.check                                                                \
+                  tests/specs/bihu/functional-spec.k.prove                                                     \
+                  tests/specs/examples/empty-bin-runtime.k                                                     \
+                  tests/specs/examples/erc20-bin-runtime.k                                                     \
                   tests/specs/examples/erc721-bin-runtime.k
 
 test-kevm-pyk: KPROVE_OPTS  += --pyk --verbose
 test-kevm-pyk: KOMPILE_OPTS += --pyk --verbose
 test-kevm-pyk: KAST_OPTS += --pyk --verbose
+test-kevm-pyk: TEST_OPTIONS += --pyk --verbose
```

Comment:
Why is this called TEST_OPTIONS? The rest have tool names, except for this one.

Reply:
Good call. I renamed `TEST_OPTIONS` to `KEVM_OPTIONS`, and now that's the only variable that needs to be set. I've also threaded that variable through ot all the places it should be present, and added variables `KRUN_OPTS`, `KAST_OPTS`, and `KSEARCH_OPTS`.